src/app/containers/Bulletin/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useContext } from 'react';
+import pathOr from 'ramda/src/pathOr';
+import { shape, oneOfType } from 'prop-types';
+import Image from '@bbc/psammead-image';
+import Bulletin from '@bbc/psammead-bulletin';
+import { tvBulletinItem, radioBulletinItem } from '#models/propTypes/bulletin';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const BulletinContainer = ({ item }) => {
+  const { script, service, dir } = useContext(ServiceContext);
+
+  const contentType = pathOr(null, ['contentType'], item);
+  const isLive = pathOr(null, ['isLive'], item);
+  const headline = pathOr(null, ['name'], item);
+  const summary = pathOr(null, ['summary'], item);
+  const ctaLink = pathOr(null, ['uri'], item);
+  const imageSrc = pathOr(null, ['indexImage', 'href'], item);
+  const imageAlt = pathOr(null, ['indexImage', 'altText'], item);
+  const bulletinImage = <Image src={imageSrc} alt={imageAlt} />;
+
+  const ctaText = contentType === 'TVBulletin' ? 'Watch' : 'Listen';
+  const type = contentType === 'TVBulletin' ? 'video' : 'audio';
+  const offScreenText = isLive ? `${ctaText} LIVE` : ctaText;
```

Comment:
Since we have translations in the service configs it would be nice if we can pass in the translations.

Reply:
Yeah, working on it atm

Reply:
It looks like you missed the `Live` translations.

Reply:
Not sure about this.. in the JIRA ticket we have: 
`and the ‘Live’ label should be read in English, with an English accent`

Reply:
What I think we should translate and pass to the Bulletin is the LIVE we display in the headline.

Reply:
Yeah. you are right. Apart from that this looks good.

Reply:
This PR looks really good - just need to agree on what we're doing with the 'live' translation. As far as I'm aware, we still need to pass the `liveText` prop in with the translated value of 'Live', which is used in the headline - and then pass in `ctaText` with the same translation like in the psammead story:

```
const playCtaText = isLive ? `${ctaText} Live` : ctaText;
```

Reply:
As is, LIVE in the headline is always in english, and the CTA does not change

Reply:
Added the `playCtaText`. Waiting for product to confirm about the live text.

Reply:
They have decided to keep it in English for now until they review with Editorial later on.